### ISSUE TITLE
Question: Would you meet the output of LTU is poor quality in the early stage？

### ISSUE THREAD

**Issue by Kurt232** [YuanGongND/ltu#46]
I am trying to use the familiar framework to align LLama with other time-series data.
But my finetuned model rarely output formatted answer for multi-choices question, therefore it's very difficult to evaluate the model.
Would you meet the same question in the early stage of LTU?
I will appreciate your answer very much if you can share your detailed method or relevant materials.

**Prompt**: `... Your answer should follow the format: 'Answer: <a label in above options>
Reason: <optional reason, less than 30 words>'`

**Response**: `Answer: Multi-choice question about XXX: Option A - XXX. Reason: XXX. Option B - XXX. Reason: XXX`
**Response**: `"[0.65, 0.25, 0.15, 0.65, 0.45, 0.85, 0.75]\n\n### Confusion: [0.65, 0.25, 0.15, 0.65, 0.45, 0.85, 0.75]\n\n\nThe confusion values indicate that the model has difficulty in distinguishing between activities 3, 4, 5, and 6. The model's predictions are close to the real values for activities 1, 2, and 6, but not as accurate for activities 3, 4, and 5.`

by the way, my toy data scale is very tiny, it's nearly 9k, and my encoder parameter is nearly 10K. And my training hyperparameters are 5 epochs and 1e-4 learning rate
![W B Chart 2024_9_4 14_57_58](https://github.com/user-attachments/assets/8e4835ff-d7cd-42b7-b710-a0548014f925)

Thanks your time and attention.

**Comment by YuanGongND** [comment]
this is a question that I am not sure. you may need to check the data demand to post-train a model for a certain demand. My guess you need a pretrained encoder and llm, and probably also trained with our openaqa dataset, and then finetune on your 9k data.

-Yuan
